app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -546,11 +516,37 @@ open class DefaultToolbarMenu(
     val deleteDataOnQuit = BrowserMenuImageText(
         label = context.getString(R.string.delete_browsing_data_on_quit_action),
         imageResource = R.drawable.ic_exit,
-        iconTintColorResource = primaryTextColor
+        iconTintColorResource = primaryTextColor()
     ) {
         onItemTapped.invoke(ToolbarMenu.Item.Quit)
     }
 
+    val syncedTabsItem = BrowserMenuImageText(
+        context.getString(R.string.synced_tabs),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor()
+    ) {
+        onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)
+    }
+
+    private val syncSignInText = context.getString(R.string.sync_menu_sign_in)
+    private fun getSyncItemTitle(): String {
```

Comment:
I suspect you're gonna use this in https://github.com/mozilla-mobile/fenix/pull/19037. Can we move this to any general Util file that can be used between the 2? Also, a comment for the function would be nice.

Reply:
There's a refactoring PR that I have in progress right now, I'm going to do that there

Reply:
Ok, let's make sure we make a note of this to not forget.

Reply:
It's in the other PR